README.md

```diff
@@ -58,7 +60,46 @@ Experimenter is a platform for managing experiments in [Mozilla Firefox](https:/
 
         https://localhost/
 
-Done!
+#### Semi Dockerized Setup (continuation from General Setup 1-7)
+
+Node ^14.0.0 is required
+
+[osx catalina, reinstall command line tools](https://medium.com/flawless-app-stories/gyp-no-xcode-or-clt-version-detected-macos-catalina-anansewaa-38b536389e8d)
+
+1.  Pre reqs (macOs instructions)
+
+        brew install postgresql llvm openssl yarn
+
+        echo 'export PATH="/usr/local/opt/llvm/bin:$PATH"' >> ~/.bash_profile
+        export LIBRARY_PATH=$LIBRARY_PATH:/usr/local/opt/openssl/lib/
+
+2.  Install dependencies (at root)
+
+        source .env
+
+        poetry install (cd into app for this)
+
+        yarn install
+
+3.  env values (at root)
+
+        .env:
+        DEBUG=True
+        POSTGRES_HOST=localhost
+        HOSTNAME=localhost
+        KINTO_HOST=http://localhost:8888/v1
+        REDIS_HOST=localhost
+
+4.  Start postgresql, redis, autograph, kinto (at root)
+
+        make up_db
+
+5.  Django app (at root)
+
+        poetry shell
+        yarn workspace @experimenter/core build
+        yarn workspace @experimenter/rapid build
+        ./manage.py runserver 0.0.0.0:7001
```

Comment:
Hm, this resulted in this series of errors: https://gist.github.com/jodyheavener/136ff82753f1a49db779f00ecc69347c

Reply:
oh silly me it's DB_HOST not POSTGRES_HOST, lemme make those changes...